aio/deploy/helm-chart/kubernetes-dashboard/templates/ingress.yaml

```
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{{- if .Values.ingress.enabled -}}
+{{- if .Values.ingress.enabled }}
```

This fix doesn't work, because the whitespace is still removed by the next tag. The correct fix is:
```suggestion
{{ if .Values.ingress.enabled -}}
```

---

I'm going to add several tests in CI later today. It should solve once and for all those issues.

---

Here we are, along with the correct fixes: https://travis-ci.org/wiremind/dashboard/jobs/645872825